app/models/importers/base_importer.rb

```diff
@@ -109,7 +109,18 @@ def import(path)
     records_report("initial")
 
     rows = CSV.read(path, headers: true)
+    import_rows(rows)
+  end
+
+  def import_string(string)
+    Rails.logger.info("START IMPORT------------#{Time.now}")
+    records_report("initial")
+
+    rows = CSV.parse(string, headers: true)
+    import_rows(rows)
+  end
```

Comment:
Not sure if you want different logging in these two alternate methods. Just kept it consistent with how it currently works.

Ideally, we could rename `import` as well, but i didn't want to break the existing interface.

Reply:
love it!!! THANK YOU!!!!!